.github/workflows/ci.yml

```diff
@@ -43,6 +43,7 @@ jobs:
           MODULE: ${{env.MODULE}}
           dpf-standalone-TOKEN: ${{secrets.DPF_PIPELINE}}
           install_extras: plotting
+          wheelhouse: "false"
```

Comment:
@PProfizi is that meant to stay?

Reply:
@cbellot000 yes, meant to stay because we do not want to generate a full wheelhouse every time.